rest-api/dao/questionnaire_response_dao.py

```diff
@@ -229,6 +239,11 @@ def _update_participant_summary(
           new_status = QuestionnaireStatus.SUBMITTED
           if code.value == CONSENT_FOR_ELECTRONIC_HEALTH_RECORDS_MODULE and not ehr_consent:
             new_status = QuestionnaireStatus.SUBMITTED_NO_CONSENT
+          if code.value == CONSENT_FOR_DVEHR_MODULE:
+            new_status = dvehr_consent
+          if code.value == CONSENT_FOR_DVEHR_MODULE and \
```

Comment:
You don't need this if anymore I believe.

Reply:
done